dae/dae/configuration/schemas/dae_conf.py

```diff
@@ -50,6 +50,11 @@
     "password": {"type": "string"},
 }
 
+gsd_schema = {
+    "id": {"type": "string"},
+    "url": {"type": "string"}
```

Comment:
probably it will be good to have "name" and "description" for each of the repositories